src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -10,6 +10,7 @@ import useChatClientHook from '../../../utils/getstream/useChatClientHook';
 import {COLORS} from '../../../utils/theme';
 import useDiscovery from '../../DiscoveryScreenV2/hooks/useDiscovery';
 import UserItem from './UserItem';
+import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useRootChannelListHook` is added, which seems to be used for managing the channel list. This change looks good as it's necessary for the new functionality.
```diff
+ import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->